tests/test_facebook_attribution_window.py

```diff
@@ -27,35 +24,50 @@ def get_properties(self, original: bool = True):
         return_value["start_date"] = self.start_date
         return return_value
 
+    # For the test ad set up in facebook ads manager we see data on April 7th, start date is based on this data
     def test_run(self):
-        self.run_test(self.ATTRIBUTION_WINDOW) # attribution window: 7
+        # attrribution window = 7
+        self.ATTRIBUTION_WINDOW = 7
+        self.start_date = '2021-04-14T00:00:00Z'
+        self.end_date = '2021-04-15T00:00:00Z'
+        self.run_test(self.ATTRIBUTION_WINDOW, self.start_date, self.end_date)
 
+        # attribution window = 28
         self.ATTRIBUTION_WINDOW = 28
-        self.run_test(self.ATTRIBUTION_WINDOW) # attribution window: 28
+        self.start_date = '2021-04-30T00:00:00Z'
+        self.end_date = '2021-05-01T00:00:00Z'
+        self.run_test(self.ATTRIBUTION_WINDOW, self.start_date, self.end_date)
+
+        # attribution window = 1
+        self.ATTRIBUTION_WINDOW = 1
+        self.start_date = '2021-04-08T00:00:00Z'
+        self.end_date = '2021-04-09T00:00:00Z'
+        self.run_test(self.ATTRIBUTION_WINDOW, self.start_date, self.end_date)
 
-    def run_test(self, attr_window):
+
+    def run_test(self, attr_window, start_date, end_date):
```

Comment:
Not really an issue, but there it's not necessary to pass these arguments into the method since they are all accessible as class variables.